src/resources/dicts/de.edn

```diff
@@ -20,11 +20,9 @@
  :export-page "Seite exportieren"
  :export-public-pages "Öffentliche Seiten exportieren"
  :export-roam-json "Als Roam JSON exportieren"
- :graph "Graph"
```

Comment:
Romance languages do use the same English word from time to time so I'm bringing some of these back that I've confirmed are correct. There's an allowed duplicates config in lang.clj for these exceptions

Reply:
sounds good. The issue I ran into with this is where to set the line? For example many Germanic languages use 'code' since there is no equivalent for it in many languages.

Reply:
See https://github.com/logseq/logseq/blob/master/docs/contributing-to-translations.md#fix-mistakes